--- gambit_out/AssetMappings.sol	2023-07-25 13:23:33
+++ gambit_out/mutants/85/vmex/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol	2023-07-17 01:51:50
@@ -219,7 +219,8 @@
             currentAssetMapping.supplyCap = inputAsset.supplyCap;
             currentAssetMapping.borrowCap = inputAsset.borrowCap;
             currentAssetMapping.baseLTV = inputAsset.baseLTV;
-            currentAssetMapping.liquidationThreshold = inputAsset.liquidationThreshold;
+            /// AssignmentMutation(`inputAsset.liquidationThreshold` |==> `0`) of: `currentAssetMapping.liquidationThreshold = inputAsset.liquidationThreshold;`
+            currentAssetMapping.liquidationThreshold = 0;
             currentAssetMapping.liquidationBonus = inputAsset.liquidationBonus;
             currentAssetMapping.borrowFactor = inputAsset.borrowFactor;
             currentAssetMapping.VMEXReserveFactor = inputAsset.VMEXReserveFactor;
\ No newline at end of file
