--- gambit_out/AssetMappings.sol	2023-07-25 13:23:33
+++ gambit_out/mutants/110/vmex/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol	2023-07-17 01:51:50
@@ -238,7 +238,8 @@
             else {
                 // add to end
                 assetMappings[approvedAssetsTail].nextApprovedAsset = currentAssetAddress;
-                approvedAssetsTail = currentAssetAddress;
+                /// DeleteExpressionMutation(`approvedAssetsTail = currentAssetAddress` |==> `assert(true)`) of: `approvedAssetsTail = currentAssetAddress;`
+                assert(true);
             }
 
             emit AssetDataSet(
\ No newline at end of file
