--- gambit_out/AssetMappings.sol	2023-07-25 13:23:33
+++ gambit_out/mutants/105/vmex/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol	2023-07-17 01:51:50
@@ -226,7 +226,8 @@
             currentAssetMapping.borrowingEnabled = inputAsset.borrowingEnabled;
             currentAssetMapping.assetType = inputAsset.assetType;
             currentAssetMapping.isAllowed = true;
-            currentAssetMapping.exists = true;
+            /// AssignmentMutation(`true` |==> `false`) of: `currentAssetMapping.exists = true;`
+            currentAssetMapping.exists = false;
 
             interestRateStrategyAddress[currentAssetAddress][0] = inputAsset.defaultInterestRateStrategyAddress;
 
\ No newline at end of file
