--- gambit_out/AssetMappings.sol	2023-07-25 13:23:33
+++ gambit_out/mutants/180/vmex/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol	2023-07-17 01:51:50
@@ -357,7 +357,8 @@
     }
 
     function getAssetBorrowable(address asset) view external returns (bool){
-        require(assetMappings[asset].isAllowed, Errors.AM_ASSET_NOT_ALLOWED);
+        /// RequireMutation(`assetMappings[asset].isAllowed` |==> `false`) of: `require(assetMappings[asset].isAllowed, Errors.AM_ASSET_NOT_ALLOWED);`
+        require(false, Errors.AM_ASSET_NOT_ALLOWED);
         return assetMappings[asset].borrowingEnabled;
     }
 
\ No newline at end of file
