--- gambit_out/AssetMappings.sol	2023-07-25 13:23:33
+++ gambit_out/mutants/150/vmex/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol	2023-07-17 01:51:50
@@ -283,7 +283,8 @@
         assetMappings[asset].supplyCap = (supplyCap);
         assetMappings[asset].borrowCap = (borrowCap);
         assetMappings[asset].borrowFactor = (borrowFactor);
-        assetMappings[asset].isAllowed = true;
+        /// DeleteExpressionMutation(`assetMappings[asset].isAllowed = true` |==> `assert(true)`) of: `assetMappings[asset].isAllowed = true;`
+        assert(true);
 
         emit ConfiguredAssetMapping(asset, baseLTV, liquidationThreshold, liquidationBonus, supplyCap, borrowCap, borrowFactor);
     }
\ No newline at end of file
