--- gambit_out/AssetMappings.sol	2023-07-25 13:23:33
+++ gambit_out/mutants/200/vmex/packages/contracts/contracts/protocol/lendingpool/AssetMappings.sol	2023-07-17 01:51:50
@@ -389,7 +389,8 @@
     }
 
     function getBorrowFactor(address asset) view external returns(uint256){
-        require(assetMappings[asset].isAllowed, Errors.AM_ASSET_NOT_ALLOWED); //not existing
+        /// RequireMutation(`assetMappings[asset].isAllowed` |==> `true`) of: `require(assetMappings[asset].isAllowed, Errors.AM_ASSET_NOT_ALLOWED); //not existing`
+        require(true, Errors.AM_ASSET_NOT_ALLOWED); //not existing
         return assetMappings[asset].borrowFactor;
     }
 
\ No newline at end of file
